products/sourcegraph.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Sourcegraph
+category: server-app
+tags: java-runtime
+permalink: /sourcegraph
+releasePolicyLink: https://handbook.sourcegraph.com/departments/engineering/dev/process/releases/
+releaseDateColumn: true
+eolColumn: Support
+
+identifiers:
+- purl: pkg:github/sourcegraph/sourcegraph
+- purl: pkg:docker/sourcegraph/sg
+
+auto:
+  methods:
+  - git: https://github.com/sourcegraph/sourcegraph.git
+
+releases:
+- releaseCycle: "5"
+  releaseDate: 2023-05-22
+  eol: false
+  latest: "5.3.3"
+  latestReleaseDate: 2024-03-20
+  link: https://github.com/sourcegraph/sourcegraph/releases/tag/v__LATEST__
```

Comment:
```suggestion
```

We could use this as the main `changelogTemplate` as it works for both 4.x and 5.x. See my previous comment.